pixi.toml

```diff
@@ -30,12 +30,20 @@ showcase = { cmd = "python examples/showcase.py $0"}
 click = ">=8.1.7"
 matplotlib = ">=3.9.0,<4"
 
+[feature.showcasevideo.tasks]
+showcase_video = "python examples/showcase_video.py"
+
+[feature.showcasevideo.dependencies]
+click = ">=8.1.7"
+
 [dependencies]
 max = ">=25.3.0.dev2025042905,<26"
 pillow = ">=11.0.0,<12"
+py-opencv = ">=4.11.0,<5"
```

Comment:
This dependency should probably be moved up into the `[feature.showcasevideo.dependencies]` section, so that it's only pulled in when using the video showcase example.